apps/theming-designer/src/components/Samples.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+import { Card } from '@uifabric/react-cards';
+import { SamplesCardHeader } from './SamplesCardHeader';
+import {
+  ActionButton,
+  Checkbox,
+  DefaultButton,
+  Dropdown,
+  Link,
+  PrimaryButton,
+  Stack,
+  Text,
+  TextField,
+  Toggle
+} from '../../../../packages/office-ui-fabric-react/lib';
+
+export class Samples extends React.Component {
+  render() {
+    return (
+      <Card styles={{ root: { width: '800px', height: 'auto' } }}>
+        <h1>Samples</h1>
+        <Stack horizontal gap={50}>
+          <Stack gap={10}>
+            <SamplesCardHeader label="TEXT" />
+            <SamplesCardHeader size="tiny" label="MEMBER FEATURE STORY" />
+            <Text variant="xLarge">The Mystery of Color</Text>
+            <Stack>
+              <Text variant="tiny" block>
+                Contrast is the difference in
+              </Text>
+              <Text variant="tiny" block>
+                luminance or color that makes an{' '}
+              </Text>
+              <Text variant="tiny" block>
+                {' '}
+                object (or its representation in an{' '}
+              </Text>
+              <Text variant="tiny" block>
+                {' '}
+                image or display) distinguishable.{' '}
+              </Text>
+              <Text variant="tiny" block>
+                {' '}
+                In visual perception of the real{' '}
+              </Text>
+              <Text variant="tiny" block>
+                {' '}
+                world, contrast is determined{' '}
+              </Text>
+            </Stack>
+          </Stack>
+
+          <Stack gap={10}>
+            <SamplesCardHeader label="LINKS" />
+            <Link href="http://dev.office.com/fabric/components/link">Title hyperlink</Link>
+            <Link disabled={true} href="http://dev.office.com/fabric/components/link">
+              Disabled link
+            </Link>
+            <SamplesCardHeader label="BUTTONS" />
+            <Stack horizontal>
+              <PrimaryButton text="Primary" />
+              <DefaultButton text="Standard" />
+            </Stack>
+            <SamplesCardHeader label="ACTION BUTTONS" />
+            <Stack horizontal gap={5}>
+              <ActionButton
+                data-automation-id="test"
```

Comment:
Consider cleaning up the attributes / props copy pasted throughout as it'll be difficult to remove later. One example being `data-automation-id` which I doubt is used.